src/pages/signin/signin.scss

```
@@ -0,0 +1,13 @@
+@import "../../styles/style.scss", "../../styles/auth.scss";
+
+.auth_container_form {
+	background: $bgFormColor;
+	width: 50%;
+	padding: 200px 65px 65px 65px;
```

Можно использовать "шортхэнд"-запись
```suggestion
	padding: 200px 65px 65px;
```